dpnp/tests/test_usm_type.py

```diff
@@ -815,33 +774,33 @@ def test_2in_1out(func, data1, data2, usm_type_x, usm_type_y):
         pytest.param("searchsorted", [11, 12, 13, 14, 15], 13),
     ],
 )
-@pytest.mark.parametrize("usm_type", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type", list_of_usm_types)
 def test_2in_with_scalar_1out(func, data, scalar, usm_type):
-    x = dp.array(data, usm_type=usm_type)
-    z = getattr(dp, func)(x, scalar)
+    x = dpnp.array(data, usm_type=usm_type)
+    z = getattr(dpnp, func)(x, scalar)
     assert z.usm_type == usm_type
 
 
-@pytest.mark.parametrize("usm_type", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type", list_of_usm_types)
 def test_apply_along_axis(usm_type):
-    x = dp.arange(9, usm_type=usm_type).reshape(3, 3)
-    y = dp.apply_along_axis(dp.sum, 0, x)
+    x = dpnp.arange(9, usm_type=usm_type).reshape(3, 3)
+    y = dpnp.apply_along_axis(dpnp.sum, 0, x)
 
     assert x.usm_type == y.usm_type
 
 
-@pytest.mark.parametrize("usm_type", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type", list_of_usm_types)
 def test_apply_over_axes(usm_type):
-    x = dp.arange(18, usm_type=usm_type).reshape(2, 3, 3)
-    y = dp.apply_over_axes(dp.sum, x, [0, 1])
+    x = dpnp.arange(18, usm_type=usm_type).reshape(2, 3, 3)
+    y = dpnp.apply_over_axes(dpnp.sum, x, [0, 1])
 
     assert x.usm_type == y.usm_type
 
 
-@pytest.mark.parametrize("usm_type", list_of_usm_types, ids=list_of_usm_types)
+@pytest.mark.parametrize("usm_type", list_of_usm_types)
 def test_broadcast_to(usm_type):
```

Comment:
It seems it can be moved to `test_2in_with_scalar_1out` as
`pytest.param("broadcast_to", numpy.ones(7), (2, 7)),`